src/main/resources/db.properties

```diff
@@ -0,0 +1,3 @@
+db.url=jdbc:mysql://localhost:3306/eventplannerdb
```

Comment:
You need to connect to the remote MySQL server

Reply:
I will, thank you!